utils.c

```diff
@@ -250,6 +256,7 @@ static const struct luaL_Reg torch_utils__ [] = {
   {"pointer", luaT_lua_pointer},
   {"setheaptracking", torch_setheaptracking},
   {"updateerrorhandlers", torch_updateerrorhandlers},
+  {"getmemory", torch_getmemory},
```

Comment:
`getMemoryUsage` sounds good to me as well 😄 I agree that it might be better to keep it similar.